docs/_extra/api-reference/schemas/user-new.yaml

```diff
@@ -0,0 +1,38 @@
+User:
+  type: object
+  anyOf:
+    - required:
```

Comment:
Does this mean that you need to provide (authority, username, email) OR (authority, username, identities)?

Reply:
Yes, it seems so:

<img width="498" alt="screenshot 2019-02-07 17 52 05" src="https://user-images.githubusercontent.com/2458/52431916-1f346580-2b01-11e9-81cc-cd8c5440e8a0.png">

Is it possible to display a more helpful name in the "Any of" list in that view?

Reply:
It's JSON Schema terminology; maybe not the most immediately intuitive to everyone, but technically accurate.